applications/Aisland-DocSig.md

```diff
@@ -0,0 +1,241 @@
+# Aisland Docsig
+
+- **Team Name:** AISLAND DAO
+- **Payment Address:** 0x78A4C8624Ba26dD5fEC90a8Dc9B75B4E3D630035 USDC/Ethereum
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+## Project Overview :page_facing_up:
+
+### Overview
+
+Aisland DocSig allows to e-sign documents, exchange easily and store permanently on chain as legal proof. 
+POC/MVP is available at [https://docsig.aisland.io](https://docsig.aisland.io)  
```

Comment:
I tested it but after initially signing it with the doc creator account I wasn't able to sign it with a counterparty account:

<img width="831" alt="image" src="https://github.com/w3f/Grants-Program/assets/5393704/78c478fb-f711-474b-9ef5-d8ef5da0dfca">

As you can see the status is _Waiting_ here. If I now copy the link: 

<img width="463" alt="image" src="https://github.com/w3f/Grants-Program/assets/5393704/d0d82e7b-76fc-4a4d-9f18-a04d3ff14dc7">

...and paste it in a different browser session to try to sign it it won't work. Instead I get an error when viewing it.

<img width="866" alt="image" src="https://github.com/w3f/Grants-Program/assets/5393704/ed132ab8-a01a-4791-8c5d-7a906f70ed7e">

<img width="546" alt="image" src="https://github.com/w3f/Grants-Program/assets/5393704/e168aeb9-834d-4e40-81c8-d88d011ac8f3">

Reply:
thanks for reporting the issue. We have pushed a set of changes that should have fixed the issue above.